packages/ui/src/forum/components/ForumBreadcrumbs.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import { BreadcrumbsItem } from '@/common/components/page/Sidebar/Breadcrumbs/BreadcrumbsItem'
+import { BreadcrumbsListComponent } from '@/common/components/page/Sidebar/Breadcrumbs/BreadcrumbsList'
+
+import { ForumBreadcrumb } from '../types'
+
+interface Props {
+  forumBreadcrumbs: ForumBreadcrumb[]
+  currentBreadcrumb: string
+}
+
+export const ForumBreadcrumbs = React.memo(({ forumBreadcrumbs, currentBreadcrumb }: Props) => {
+  const crumbs = forumBreadcrumbs.map((crumb) => ({
+    path: `/forum/categories/${crumb.id}`,
+    breadcrumb: crumb.title,
+    key: crumb.id + crumb.title,
+  }))
+  crumbs.push({ path: '', breadcrumb: currentBreadcrumb, key: 'last:' + currentBreadcrumb })
+  return (
+    <BreadcrumbsListComponent>
+      {crumbs.map(({ path, breadcrumb, key }, index, { length }) => (
+        <BreadcrumbsItem key={key} url={path} isLink={index < length - 1}>
+          {breadcrumb}
+        </BreadcrumbsItem>
+      ))}
+    </BreadcrumbsListComponent>
```

Comment:
Since the not link is always pushed at the end, maybe something like below:

```suggestion
    <BreadcrumbsListComponent>
      {forumBreadcrumbs.map(({ id, title }) => (
        <BreadcrumbsItem key={id} url={`/forum/categories/${crumb.id}`} isLink>
          {title}
        </BreadcrumbsItem>
      ))}
      <BreadcrumbsItem key="_current">
        {currentBreadcrumb}
      </BreadcrumbsItem>
    </BreadcrumbsListComponent>
```

The above was written on GH so it might need some adjustments. The `<BeradcrumbsItem>` should allow empty `url` when `isLink` is `undefined` (so not set).

Reply:
The other benefit here would be one loop vs two loops for rendering this component.

Reply:
I decided to make `currentBreadcrumb` optional, to make it work more like the breadcrumbs component we already have